plugins/eyetracker/com_osvr_EyeTracker.cpp

```diff
@@ -0,0 +1,141 @@
+/** @date 2015
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2015 Vuzix Corporation.
```

Comment:
I think this is mistaken - are you copy/pasting files or using the boilerplate generator?